go/cmd/dolt/dolt.go

```diff
@@ -587,9 +587,10 @@ func buildLateBinder(ctx context.Context, cwdFS filesys.Filesys, mrEnv *env.Mult
 	}
 
 	if hasUseDb {
-		targetEnv = mrEnv.GetEnv(useDb)
+		dbName, _ := dsess.SplitRevisionDbName(useDb)
+		targetEnv = mrEnv.GetEnv(dbName)
 		if targetEnv == nil {
-			return nil, fmt.Errorf("The provided --use-db %s does not exist or is not a directory.", useDb)
+			return nil, fmt.Errorf("The provided --use-db %s does not exist or is not a directory.", dbName)
```

Comment:
Is it still possible to use the `--use-db` flag to specify a directory? Is that something that we want to support? If not, we should probably edit this error message.

Reply:
Yeah, that message doesn't make sense.